.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
```

Comment:
Replace with `Support Request` (also please change md file name).

Reply:
+1 to limit the category of issues, shall we give a reminding if users want to report a bug, please go to JIRA? Another question: does "suggestion" belong to "support" here?